LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def do_linear_approach(datasets, penalties):
</pre><p>Rename to something like <code>_update_w_lp</code> to distinguish it from <code>update_w</code>   and then make sure it has matching return types.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>They do not have the same output: </p><p>_update_w_lp: list, weights for all features for all matrices</p><p>update_w: w[f] float, weights for one feature and one matrix</p><p> </p>